libsolidity/ast/Types.h

```diff
@@ -1237,6 +1237,7 @@ class FunctionType: public Type
 		ABIEncode,
 		ABIEncodePacked,
 		ABIEncodeWithSelector,
+		ABIEncodeChecked,
```

Comment:
Why don't you call it `ABIEncodeCall`?

Reply:
I thought the unique new thing about this one was that it checks the types.. thus `Checked` but I don't mind how exactly we call it...